scripts/Makefile.pdb

```diff
@@ -70,3 +77,11 @@ check-deps:
 			{ echo "Error: Dependency requirement not satisfied, Please run 'pip install -r $(PYTHON_DEP_FILE)'"; exit 1; }; \
 	done
 
+package-pdb: 
+	cp -r scripts/xspdb ${PDB_HOME}/XSPdb
```

Comment:
can we choose another `temporary directory` for package? otherwise, all modifies to `scripts/xspdb/*` after `make package-pdb` action will not take effect (because the `script` dir was placed after PDB_HOME as following)
```
PDB_PYTHONPATH    := $(PDB_HOME):script/xspdb
```